From e21001ca08c630c6ef97a7e0f9625eef4c772601 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marcus=20M=C3=BCller?= <mmueller@gnuradio.org>
Date: Sat, 17 Aug 2019 17:39:55 +0200
Subject: [PATCH 08/14] clang-tidy: Edit config to include checks that have
 been executed

Not including the modernize-use-emplace, since that really broke nearly
every line it touched.
---
 .clang-tidy | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 .clang-tidy

diff --git a/.clang-tidy b/.clang-tidy
new file mode 100644
index 0000000000..a9c77731da
--- /dev/null
+++ b/.clang-tidy
@@ -0,0 +1,10 @@
+---
+Checks:          '-*,misc-throw-by-value-catch-by-reference,misc-static-assert'
+# Not in here: modernize-use-emplace, since that basically broke all things it touched
+WarningsAsErrors: ''
+HeaderFilterRegex: '\.(cc|c|cpp|h|hpp)$'
+AnalyzeTemporaryDtors: false
+FormatStyle:     file
+CheckOptions:
+...
+
-- 
2.20.1

